### DISCUSSION TITLE
how well adopted is Otel ?

### DISCUSSION THREAD

**Discussion by melamoud** [open-telemetry/community#1294] [Community Discussions]
Do we have data on % of adoption ? 
do we have data on major users ?  and how big the scale they are implementing ?

**Comment by SergeyKanzhelev** [comment]
You may be interested to join end users SIG meeting to ask this question. Some links to start with:

- https://github.com/open-telemetry/community/tree/main/working-groups/end-user
- https://github.com/open-telemetry/community/blob/main/ADOPTERS.md
- https://github.com/open-telemetry/opentelemetry.io/blob/main/content/en/vendors.md

**Reply by melamoud** [reply]
not sure, too early for me, I want to make a decision if I will use Otel, at all, and the first question (that is not technical) ois how widely it is been used, how mature it is , at what scale  ?

**Reply by SergeyKanzhelev** [reply]
thank you for additional context. There are a few monitoring vendors made Otel a suggested or even primary source of telemetry. Which can be used as a good indication of maturity. Depending on specific telemetry signal you want to collect or framework you are using you may likely find better tools for certain signals. But from wide coverage, future proof, and integration Otel is a best bet.

**Reply by melamoud** [reply]
Thanks, Im almost there, I just need to see some numbers, Im a number guy, :) but I cant find any source that give any numbers around adoption

**Comment by tigrannajaryan** [comment]
The list of vendors that support Otel is not exactly what you asked for but may be useful: https://opentelemetry.io/vendors/
